src/components/__tests__/SecurityRequirement.test.tsx

```diff
@@ -1,28 +1,156 @@
 import * as React from 'react';
-import { shallow } from 'enzyme';
+import { mount } from 'enzyme';
 
-import { OpenAPIParser } from '../../services';
-import { SecurityRequirementModel } from '../../services/models/SecurityRequirement';
-import { SecurityRequirement } from '../SecurityRequirement/SecurityRequirement';
-import { RedocNormalizedOptions } from '../../services/RedocNormalizedOptions';
+import {
+  createStore,
+  OpenAPIParser,
+  OperationModel,
+  RedocNormalizedOptions,
+  SecuritySchemesModel,
+} from '../../services';
+import { StoreProvider } from '../StoreBuilder';
+import { SecurityRequirements } from '../SecurityRequirement/SecurityRequirement';
+import { withTheme } from '../testProviders';
+import { OpenAPISpec } from '../../types';
+import { SecurityDefs } from '../SecuritySchemes/SecuritySchemes';
 
-const options = new RedocNormalizedOptions({});
-describe('Components', () => {
-  describe('SecurityRequirement', () => {
-    describe('SecurityRequirement', () => {
-      it("should render 'None' when empty object in security open api", () => {
-        const parser = new OpenAPIParser(
-          { openapi: '3.0', info: { title: 'test', version: '0' }, paths: {} },
-          undefined,
-          options,
+const spec: OpenAPISpec = {
```

Comment:
Maybe better move spec to `fixtures`?

Reply:
Does it cover all security types, which we support?
``` 
  oauth2: 'OAuth2',
  apiKey: 'API Key',
  http: 'HTTP',
  openIdConnect: 'OpenID Connect',
```

Reply:
> Maybe better move spec to fixtures?
Done 😄